luajit/luajit.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "luajit",
+    "no-autogen": true,
+    "make-args": [
+        "PREFIX=${FLATPAK_DEST}"
+    ],
+    "make-install-args": [
+        "PREFIX=${FLATPAK_DEST}"
+    ],
+    "sources": [
+        {
+            "type": "archive",
+            "url": "https://luajit.org/download/LuaJIT-2.1.0-beta3.tar.gz",
+            "sha256": "1ad2e34b111c802f9d0cdf019e986909123237a28c746b21295b63c9e785d9c3",
+            "x-checker-data": {
+                "type": "anitya",
+                "project-id": 6444,
+                "stable-only": false,
+                "url-template": "https://luajit.org/download/LuaJIT-$version.tar.gz"
+            }
+        }
+    ],
+    "cleanup": [
+        "/bin",
+        "/include",
+        "/lib/*.a",
```

Comment:
Does this actually build a `.a` file? Can you disable that at build time?

Reply:
It does, but I'll see if that can be disabled. Plain autotools is hard to navigate 😕

Reply:
Oh I wished Luajit used autotools. No they hand roll terrible Makefiles.

Reply:
I don't see an option for it.

You can just do:

```json
"post-install": [
    "rm $FLATPAK_DEST/lib/luajit*.a"
]
```

The reason I ask this is because if the `.a` is installed then it is possible for an application to accidentally link to that one.

Reply:
> I don't see an option for it.

Just found it, if I pass `BUILDMODE=dynamic` to `make` it does not compile the static library.